tests/TestCase/ORM/Behavior/TranslateBehaviorTest.php

```diff
@@ -973,6 +973,36 @@ public function testAllowEmptyFalse()
         $this->assertEmpty($noFra);
     }
 
+    /**
+     * Tests adding new translation to a record
+     *
+     * @return void
+     */
+    public function testAllowEmptyFalseWithNull()
+    {
+        $table = $this->getTableLocator()->get('Articles');
+        $table->addBehavior('Translate', ['fields' => ['title', 'description'], 'allowEmptyTranslations' => false]);
+
+        $article = $table->find()->first();
+        $this->assertSame(1, $article->get('id'));
+
+        $article = $table->patchEntity($article, [
+            '_translations' => [
+                'fra' => [
+                    'title' => 'Title',
+                ],
+            ],
+        ]);
+
+        $table->save($article);
+
+        // Remove the Behavior to unset the content != '' condition
+        $table->removeBehavior('Translate');
+
+        $fra = $table->I18n->find()->where(['locale' => 'fra'])->first();
```

Comment:
You'll have to copy this test method to `TranslateBehaviorShadowTableTest` too and change above line to `$fra = $table->ArticlesTranslations->find()->where(['locale' => 'fra'])->first();` for the test suite to pass.

Reply:
Done.